src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/ConfigUpdater.cs

```diff
@@ -28,6 +29,7 @@ public ConfigUpdater(XDocument doc, ILogger<ConfigUpdater> logger)
         {
             _config = doc;
             _logger = logger;
+            SetServiceAndBehaviorNames(_logger);
         }
```

Comment:
You don't need to pass the ILogger, you've saved it in a field and the method is an instance method so can access it.